#include <exception>

class error_already_set : public std::exception
{
};
